mkt/search/indexers.py

```diff
@@ -352,7 +358,7 @@ def attach_language_specific_analyzers(cls, mapping, field_names):
         """
         def _locale_field_mapping(field, analyzer):
             return {
-                '%s_%s' % (field, analyzer): {
+                '%s_l_%s' % (field, analyzer): {
```

Comment:
Yeah. I'm open to suggestions - it doesn't have to be one single letter, we just need *something* that can make it easy to exclude all these fields. Maybe `_l10n_` would work, actually.